samples/node/src/config.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs')
+const util = require('util')
+const readFileAsync = util.promisify(fs.readFile)
+const ConnectedCarsApi = require('./connectedcarsapi')
+
+const createConnectedCarsApi = async () => {
+  const environment = process.env.CC_ENVIRONMENT
+
+  // Read the config however you like
+  const ccConfig = await readFileAsync(process.env.SERVICE_ACCOUNT_KEY_FILE, {
```

Comment:
I would actually use the sync version here as you don't really want to deal with promises just to consume an API